packages/entity-browser/src/components/DetailView/FieldErrorList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const FieldErrorList = props => {
+  if (!props.errors) {
+    return null
+  }
+
+  let errorValues = []
+  Object.keys(props.errors).map(key => {
+    errorValues = [...errorValues, ...props.errors[key]]
+  })
```

Comment:
Maybe it doesn't make sense to create a new `errorValues` array and throw the old one away for each errors entry. Maybe this results in a garbage collection overhead.